[system] - Add a flag to the system module to control whether metrics failures mark agent as degraded (#42160)
elastic/beats: metricbeat/docs/modules/system.asciidoc

  # Only effective for Windows.
  # You should use this option if running beats on machins with more than 64 cores.
  #use_performance_counters: false

>   # While running in unprivileged mode, process/process_summary metricsets can emit

Reviewer: ```suggestion
  # While running as an unprivileged user, process/process_summary metricsets can emit
```

Beats doesn't have an unprivileged mode.
